src/Api/Controllers/CiphersController.cs

```diff
@@ -600,8 +600,8 @@ public async Task<AttachmentUploadDataResponseModel> PostAttachment(string id, [
                 throw new BadRequestException($"Max file size is {CipherService.MAX_FILE_SIZE_READABLE}.");
             }
 
-
-            var (attachmentId, uploadUrl) = await _cipherService.CreateAttachmentForDelayedUploadAsync(cipher, request, userId);
+            var (attachmentId, uploadUrl) = await _cipherService.CreateAttachmentForDelayedUploadAsync(cipher,
```

Comment:
Does this not already have a DTO to reduce parameter numbers?

Reply:
Nop.